Changelog.md

```diff
@@ -9,6 +9,7 @@
 - Added `AuthorizationType` to `AccessTokenAuthorizable` representing request headers of `.none`, `.basic`, and `.bearer`. 
 - Added tests for `Single<Response>` operators.
 - Added `Progress` object into the response when calling progress callback on completion.
+- Added a test for progress callback when doing upload task.
```

Comment:
Yeah, I think we can skip changelog entry here.

Reply:
Line removed
I just let [atom](https://atom.io/) remove the useless space in changelog (two lines)